qiskit/visualization/counts_visualization.py

```diff
@@ -61,6 +61,7 @@ def plot_histogram(
     figsize=None,
     color=None,
     number_to_keep=None,
+    ignore_under=None,
```

Comment:
Personally I didn't know what to expect from `ignore_under` before reading the docs, maybe we can find something a bit more descriptive... how about e.g. `count_threshold`? Suggestions welcome 🙂